patches/chrome-installer-mini_installer-chrome.release.patch

```diff
@@ -11,16 +11,22 @@ index bdf0219561c2e52fbe2e16407ac0fa9ae2e3ef30..cda55a0113704a0fd6bf719b1b785a0b
  chrome_proxy.exe: %(ChromeDir)s\
  #
  # Chrome version dir assembly manifest.
-@@ -19,6 +19,8 @@ chrome_proxy.exe: %(ChromeDir)s\
+@@ -18,9 +18,14 @@ chrome_proxy.exe: %(ChromeDir)s\
+ #
  # Chrome version dir entries, sorted alphabetically.
  #
++brave.exe.sig: %(VersionDir)s\
```

Comment:
there is no chrome.exe.sig so why do we need brave.exe.sig?

Reply:
I assume chrome has internal patch for that because they also refers sig file in version dir at https://cs.chromium.org/chromium/src/chrome/common/media/cdm_host_file_path.cc?l=54